src/app/shared/utils/count-to.directive.ts

```diff
@@ -103,6 +104,6 @@ export class CountToDirective implements OnDestroy, OnInit {
   }
 
   ngOnDestroy(): void {
-      this.ngUnsubscribe.complete();
+    this.ngUnsubscribe.complete();
```

Comment:
I think we need a `.next()` before the `.complete()` here.

Reply:
fixed in `4242c7a`